src/app.js

```diff
@@ -26,6 +24,8 @@ import Local from './local'
 import Remote from './remote'
 import Sync from './sync'
 
+// TODO: App.Permissions v3
+// eslint-disable-next-line no-unused-vars
```

Comment:
The object was used in registration, which I temporarily disabled.
I decided to keep it in order not to forget to set up permissions correctly when bringing back registration. Hence the `TODO`, and the `eslint-*` directive so eslint does complain about unused object.

I agree putting `TODO` notices everywhere is not the best, but I first thought it could be an easy way to keep track of things to port to the v3 stack.